### PR TITLE
Kickoff project base

### PR DESCRIPTION
**Pull request by wleoguty91**
## Summary

- Kickoff java project

## Trello Card

- https://trello.com/c/qxI7MtIJ/16-creacion-del-proyecto-base

Screenshot from 2019-12-05 16-40-29

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter'
+	testImplementation('org.springframework.boot:spring-boot-starter-test') {
+		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+	}
+	compile('org.springframework.boot:spring-boot-starter-data-jpa')
+	compile('org.springframework.boot:spring-boot-starter-web')
+	compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
```
Comment (facucompe) [inline build.gradle]:
```suggestion
	compile ('org.postgresql:postgresql:42.2.5')
```
You can use the same syntax you use for the other dependencies

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/dbtraining
+spring.datasource.username= java-training-postgres
+spring.datasource.password= java-training-postgres
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+server.port = 8081
```
Comment (facucompe) [inline src/main/resources/application.properties]:
```suggestion
server.port = 8081

```

**Review by facucompe — DISMISSED** [review]
Nice work

File: build.gradle (hunk 1 above)
Comment (wleoguty91) [reply build.gradle]: Oooo I don't saw that! Thanks!!

**Review by facucompe — APPROVED** [review]

**Pull request merged**